cfg/examples/replication/dex-net_4.0_fc_suction.yaml

```diff
@@ -0,0 +1,34 @@
+policy:
+  type: fully_conv_suction
+
+  sampling_method: top_k
+  gqcnn_stride: 4
+  gqcnn_recep_h: 96
+  gqcnn_recep_w: 96
+
+  # filtering params
+  max_grasps_to_filter: 50
+  filter_grasps: 0
+
+  # metrics
+  metric:
+    type: fcgqcnn
+    gqcnn_model: models/FCGQCNN-4.0-SUCTION
```

Comment:
How does one download these models? Are these just placeholders?

Reply:
Yes, they are just placeholders for now. I am waiting to upload the FC models to the Box.